content/post/jamme-jellyfish-release-notes/index.md

```diff
@@ -66,61 +66,63 @@ ISO나 플래시 가능한 이미지 다운로드:
+* 우분투 데스크톱이 다른 세대의 하드웨어에서 롤링 HWE 커널 (`linux-hwe-22.04`)을 사용합니다. 롤링 HWE 커널은 22.04와 22.04.1 포인트 릴리즈의 [v5.15](https://kernelnewbies.org/Linux_5.15) 커널에 기반합니다
+* 우분투 서버는 기본적으로 비롤링 LTS 커널인 v5.15 (`linux-generic`)에 맞춰집니다.
+* 우분투 Cloud와 Devices는 파트너사의 협력으로 최적화된 커널을 사용합니다 (추가 백포트와 기능과 더불은 v5.15+)
 
-Additional optimized and certified kernel flavours will become available in Ubuntu 22.04 LTS in due course.
+추가적 최적화와 인증된 커널 flavour는 적기에 우분투 22.04 LTS에서 이용 가능할 예정입니다.
 
-### NVIDIA drivers
+### NVIDIA 드라이버
 
-Ubuntu 22.04 LTS adds linux-restricted-modules of NVIDIA drivers on ARM64, in addition to the existing x86_64. Users on ARM64 can now use `ubuntu-drivers` tool to install and configure NVIDIA drivers from the Ubuntu Archive.
+우분투 22.04 LTS에 ARM64 NVIDIA 드라이버의 리눅스 제한 모듈이 현존 x86_64와 더불어 추가 되었습니다. 이제 ARM64 사용자는 NVIDIA 드라이버의 설치 및 설정을 위해서 `ubuntu-drivers` 도구를 우분투 아카이브에서 사용 가능합니다.
 
-### UDP disabled for NFS mounts
-Since Ubuntu 20.10 ("Groovy Gorilla"), the kernel option `CONFIG_NFS_DISABLE_UDP_SUPPORT=y` is set and this disables using UDP as the transport for NFS mounts, regardless of NFS version.
+### UDP로의 NFS 마운트 비활성화
+우분투 20.10 ("Groovy Gorilla")부터 커널 옵션 `CONFIG_NFS_DISABLE_UDP_SUPPORT=y`가 설정 됐고 이는 NFS 버전에 상관없이 UDP를 NFS 마운트의 전송 프로토콜로 사용하는 것을 비활성화 합니다.
 
-In practice, if you try to use `udp`, you will get this error:
+실제로 `udp`를 사용하려고 하면 다음과 같은 오류를 얻을 겁니다:
 ```
 $ sudo mount f1:/storage /mnt -o udp
 mount.nfs: an incorrect mount option was specified
 ```
 
 
-## Toolchain Upgrades :hammer_and_wrench:
+## 툴체인 업그레이드 :hammer_and_wrench:
 
-GCC was updated to the 11.2.0 release, binutils to 2.38, and glibc to 2.35. Python :snake: now ships at version 3.10.4, Perl :camel: at version 5.34.0. LLVM now defaults to version 14. golang defaults to version 1.18.x. rustc defaults to version 1.58.
+GCC가 11.2.0로, binutils가 2.38로, glibc가 2.35로 업데이트 됐습니다. Python :snake:은 이제 3.10.4에서, Perl :camel:은 5.34.0에서 제공됩니다. 이제 LLVM은 기본적으로 14 버전에 맞춰집니다. golang은 기본적으로 1.18.x 버전에 맞춰집니다. rustc는 기본적으로 1.58 버전에 맞춰집니다.
 
-In addition to OpenJDK 11, OpenJDK 18 is now provided (but not used for package builds).
+OpenJDK 11과 더불어 이제 OpenJDK 18도 제공됩니다 (그러나 패키지 빌드에는 사용되지 않습니다).
 
-Ruby :gem: was updated from v2.7.4 to v3.0.
+Ruby :gem:가 v2.7.4에서 v3.0로 업데이트 됐습니다.
 
 ## systemd v249.11
-The init system was updated to systemd v249, using a solid .11 patchlevel for the LTS. Please refer to the upstream [changelog](https://github.com/systemd/systemd/releases/tag/v249) for more information about the individual features. We've enabled the userspace OOMD service and are shipping the `systemd-oomd` package by default on the "Ubuntu Desktop" flavour, to avoid overloaded systems and the need of the kernel's OOM killer to kick in. The OOMD status can be checked using `oomctl`.
+init 시스템이 systemd v249로 업데이트 되었고 LTS를 위해 견고한 .11 패치 레벨이 사용됩니다. 각각 개별 기능에 대한 정보는 업스트림 [변경 로그](https://github.com/systemd/systemd/releases/tag/v249)를 참고 하길 바랍니다.

```

Comment:
We've enabled the userspace OOMD service and are shipping the `systemd-oomd` package by default on the "Ubuntu Desktop" flavour, to avoid overloaded systems and the need of the kernel's OOM killer to kick in. The OOMD status can be checked using `oomctl`

위 부분에 대한 번역이 누락되어 있습니다.

Reply:
죄송합니다. 부주의 했네요

Reply:
번역 추가 했는데 조금 긴가민가 합니다.
한번 더 체크 부탁드립니다.